src/Core/Services/Implementations/StripePaymentService.cs

```diff
@@ -749,16 +749,15 @@ private async Task ChangeOrganizationSponsorship(Organization org, OrganizationS
         prorationDate ??= DateTime.UtcNow;
         var collectionMethod = sub.CollectionMethod;
         var daysUntilDue = sub.DaysUntilDue;
-        var chargeNow = collectionMethod == "charge_automatically";
         var updatedItemOptions = subscriptionUpdate.UpgradeItemsOptions(sub);
 
         var subUpdateOptions = new Stripe.SubscriptionUpdateOptions
         {
             Items = updatedItemOptions,
-            ProrationBehavior = "always_invoice",
+            ProrationBehavior = "create_prorations",
```

Comment:
⛏️ Small nitpick here since there are only two usages of `ProrationBehavior`. Avoid using magic strings whenever possible. Separating this out into a constants file where the behavior can be explained in an XMLdoc comment would be preferable.